routes/web.php

```diff
@@ -14,45 +14,11 @@
 Route::group([
     'middleware' => 'setGlobals'
 ], function () {
-  //Route::get('/Profile', 'ProfileController@show');
-/*
-  Route::view('/Profiledata', 'Profile.home');
-  Route::get('/Profile', 'ProfileController@show');
-  Route::view('/Profile/FriendsFoes', 'Profile/FriendsFoes/home');
-  Route::view('/Profile/Heroes/All', 'Profile/Heroes/All/home');
-  Route::view('/Profile/Heroes/Single', 'Profile/Heroes/Single/home');
-
-
-
-
-
-Route::get('login', 'LoginController@redirectToProvider');
-Route::get('login/success', 'LoginController@handleProviderCallback');
-
-
-  Route::get('/Global/Talents/Builds', 'HeroController@getHeroBuildsTableData');
-  //Route::view('/', 'index');
-  Route::get('/Global/Hero/', 'HeroController@show');
-
+  Route::view('/', 'layouts.app');
 
-  */
-  Route::view('/Global/Leaderboard', 'Global/leaderboard');
-  Route::view('/Global/Hero/Talents', 'Global/Hero/talents');
+  //This route gives the correct object/data to use for the replacement code for getHeroStatsTableData.
+  //But the return is likely not what is expected so not changing it yet
   Route::view('/Global/Hero', 'Global/Hero/test');
```

Comment:
//This route gives the correct object/data to use for the replacement code for getHeroStatsTableData.
  //But the return is likely not what is expected so not changing it yet